packages/google_maps_flutter/google_maps_flutter/README.md

```diff
@@ -30,86 +39,72 @@ For more details, see [Getting started with Google Maps Platform](https://develo
-#include "GeneratedPluginRegistrant.h"
-#import "GoogleMaps/GoogleMaps.h"
-
-@implementation AppDelegate
-
-- (BOOL)application:(UIApplication *)application
-    didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-  [GMSServices provideAPIKey:@"YOUR KEY HERE"];
-  [GeneratedPluginRegistrant registerWithRegistry:self];
-  return [super application:application didFinishLaunchingWithOptions:launchOptions];
-}
-@end
-```
-
-Or in your swift code, specify your API key in the application delegate `ios/Runner/AppDelegate.swift`:
-
-```swift
-import UIKit
-import Flutter
-import GoogleMaps
-
-@UIApplicationMain
-@objc class AppDelegate: FlutterAppDelegate {
-  override func application(
-    _ application: UIApplication,
-    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
-  ) -> Bool {
-    GMSServices.provideAPIKey("YOUR KEY HERE")
-    GeneratedPluginRegistrant.register(with: self)
-    return super.application(application, didFinishLaunchingWithOptions: launchOptions)
-  }
-}
-```
+1. Specify your API key in the application delegate `ios/Runner/AppDelegate.m`:
+
+   ```objectivec
+   #include "AppDelegate.h"
+   #include "GeneratedPluginRegistrant.h"
+   #import "GoogleMaps/GoogleMaps.h"
+
+   @implementation AppDelegate
+
+   - (BOOL)application:(UIApplication *)application
+       didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+     [GMSServices provideAPIKey:@"YOUR KEY HERE"];
+     [GeneratedPluginRegistrant registerWithRegistry:self];
+     return [super application:application didFinishLaunchingWithOptions:launchOptions];
+   }
+   @end
+   ```
+
+   Or in your Swift code, specify your API key
+   in the application delegate `ios/Runner/AppDelegate.swift`:
+
+   ```swift
+   import UIKit
+   import Flutter
+   import GoogleMaps
+
+   @UIApplicationMain
+   @objc class AppDelegate: FlutterAppDelegate {
+     override func application(
+       _ application: UIApplication,
+       didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
+     ) -> Bool {
+       GMSServices.provideAPIKey("YOUR KEY HERE")
+       GeneratedPluginRegistrant.register(with: self)
+       return super.application(application, didFinishLaunchingWithOptions: launchOptions)
+     }
+   }
+   ```
+
+2. Read about iOS-specific features and limitations in the
+   [`google_maps_flutter_ios` README](https://pub.dev/packages/google_maps_flutter_ios).
 
 ### Web
 
-You'll need to modify the `web/index.html` file of your Flutter Web application
-to include the Google Maps JS SDK.
+1. Add the following to the `<head>` section of `web/index.html`:
+
+   ```html
+   <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
+   ```
 
-Check [the `google_maps_flutter_web` README](https://pub.dev/packages/google_maps_flutter_web)
-for the latest information on how to prepare your App to use Google Maps on the
-web.
+2. Read about iOS-specific features and limitations in the
```

Comment:
@filiph
I know this PR is merged already, but if you still work with documentation this could be fixed on next PR.
This should be `web` specific in this context.

Also [changelog entry ](https://github.com/flutter/packages/pull/8560/files#diff-125e4c4c56d466e14364386e43fc727a35e14f9874bb14407d8703e26fe7ce8aR8)was for old version was changed on this PR as well.

Reply:
Good catch, thanks! Opened https://github.com/flutter/packages/pull/8754.